CWE/events/italy.txt

```diff
@@ -3130,11 +3071,145 @@ country_event = {
 
 	option = {
 		name = EVT_8007762_A
-		clr_country_flag = italy_hot_autumn
-		add_country_modifier = { name = years_of_lead duration = -1 }
-		any_pop = { militancy = 3 consciousness = 3 }
+		
+		any_pop = { 
+		    militancy = 2 
+		    consciousness = 3 
+		}
+		
 		726 = { add_province_modifier = { name = terrorist_chaos duration = 365 } } #Milan
 		add_country_modifier = { name = recent_terrorist_attack duration = 183 }
+		add_country_modifier = { name = years_of_lead duration = -1 }
+		
+		set_country_flag = ITA_years_of_lead
+	}
+	
+}
+
+#Worker's Statute
+country_event = {
+ 	id = 8020340
+  	title = "Law n.300 'Worker Statute'"
+ 	desc = EVT_8020340_DESC
+  	picture = "italy_hot_autumn"
+	
+  	fire_only_once = yes
+
+	trigger = {
+		tag = ITA
+		year = 1970 
+		NOT = { year = 1990 }
+		
+		OR = { government = democracy government = democracy1 government = hms_government }
+		
+		war = no
+		is_vassal = no
+		
+		#Socialists & Progressives are not in power
+		NOT = { OR = { ruling_party_ideology = communist ruling_party_ideology = communist_social ruling_party_ideology = socialist ruling_party_ideology = progressive } }
+		
+		OR = {
+			AND = {
+				average_militancy = 9
+				has_country_modifier = italy_hot_autumn
+			}
+			AND = {
+			    any_owned_province = { controlled_by = REB }
+				has_country_modifier = italy_hot_autumn
+			}
+			AND = {
+			    revolt_percentage = 0.50
+				has_country_modifier = italy_hot_autumn
+			}
+		}
+	}
+	
+	mean_time_to_happen =  {
+		months = 1
+		modifier = {
+			factor = 0.5
+			has_country_modifier = ITA_years_of_lead
+		}
+		modifier = {
+			factor = 2
+			NOT = { has_country_modifier = ITA_years_of_lead }
+		}
+		modifier = {
+			factor = 0.8
+			NOT = { union_rights = free_trade_unions }
+		}
+		modifier = {
+			factor = 0.8
+			NOT = { wage_and_work_hours = paid_sick_leave_hours }
+		}
+		modifier = {
+			factor = 0.8
+			average_militancy = 8
+		}
+		modifier = {
+			factor = 0.8
+			average_militancy = 9
+		}
+		modifier = {
+			factor = 0.6
+			revolt_percentage = 0.6
+		}
+		modifier = {
+			factor = 0.6
+			revolt_percentage = 0.7
+		}
+		modifier = {
+			factor = 0.6
+			revolt_percentage = 0.8
+		}
+	}
+  
+	immediate = { 
+		clr_country_flag = italy_hot_autumn
+		set_country_flag = ITA_Tradeunions_Federation
+	}
+
+	option = {
+		name = "Pass the law"
+		wage_and_work_hours = paid_sick_leave_hours 
```

Comment:
Should be social_reform = X in the effects bracket and wage_and_work_hours = X in the trigger bracket